### ISSUE TITLE
Double Page Mode

### ISSUE THREAD

**Issue by Rubberxgum** [KotatsuApp/Kotatsu#649]
### Describe your suggested feature

It would be beneficial to introduce a double-page mode for manga, allowing the simultaneous display of two pages per slide. It should replicate the experience of reading a physical manga, where two pages are visible side by side, similar to Tachiyomi.

Additionaly, there should be an option to "shift one page over" in case the app groups two pages together that should not be adjacent. 

It is important to note that this feature should exclude horizontally aligned pages, as such pages are designed to be viewed as two pages already. 

The feature should look like this:
![Screenshot_20240114-095253](https://github.com/KotatsuApp/Kotatsu/assets/90501519/efd4c5df-7d62-4af3-824f-817de2e1a392)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by radry** [comment]
+1

**Comment by DonBureq** [comment]
+1

**Comment by skepsun** [comment]
It is better to determine whether to use two-page view by detecting the height-width ratio instead of detecting landscape mode. Because the two-page view is better than single-page even in portrait mode, when using some fold phones like z fold4, oneplus open.

**Comment by sho052** [comment]
Agreed. I consider this a must for any decent manga reader. Still using TachiyomiJ2K for this reason.
